src/woo_publications/metadata/tests/test_information_category_admin.py

```diff
@@ -120,7 +121,16 @@ def test_information_category_admin_can_not_update_item_with_oorsprong_waardenli
         self.assertEqual(response.status_code, 200)
 
         form = response.forms["informationcategory_form"]
+        # zelf_toegevoegd specific edible fields:
```

Comment:
```suggestion
        # zelf_toegevoegd specific editable fields:
```
you can try to eat them though